### ISSUE TITLE
training on Squad

### ISSUE THREAD

**Issue by telukuntla** [dmis-lab/biobert#19]
Hello @jhyuklee ,
May I know what are the parameters you have used while training it on Squad. like epochs and learning rate..

**Comment by jhyuklee** [comment]
We followed the default settings in the BERT repository for training SQuAD model.

**Comment by telukuntla** [comment]
Thank You, May I know is it possible for you to share the BioBert model trained on Squad

**Comment by jhyuklee** [comment]
Sorry @telukuntla , we do not provide fine-tuned weights. You have to train it yourself. Thank you.

**Comment by telukuntla** [comment]
Got it, Thanks
